content/en/guides/models/track/runs/color-code-runs.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Create semantic legends for charts
+menu:
+  default:
+    identifier: color-code-runs
+    parent: what-are-runs
+title: Semantic run plot legends
+---
+
+Create meaningful legends for run line plots to visually distinguish them based on metrics you log to W&B. Identify trends and patterns based on the highest, lowest, or latest values of a metric across your training and evaluation runs. Runs are bucketed and colored in plots based on the plot's configured metric, Y value, and number of buckets. 
+
+Navigate to your workspace's settings page to configure the metric-based colors for runs:
+
+1. Navigate to your W&B project.
+2. Select the **Workspace** tab from the project sidebar.
+3. Click on the **Settings** icon (⚙️) in the top right corner.
+4. From the drawer, select **Runs** then select **Metric-based colors**.
+    - From the **Metric** dropdown, select the metric you want to use for assigning colors to runs.
+    - From the **Y value** dropdown, select the y value you want to use for assigning colors to runs.
+    - Set the the number of buckets to a value from 2 to 8.
```

Comment:
```suggestion
    - Set **Number of buckets** to a value from 2 to 8. Optionally, click the color picker for a bucket to customize its color.
```
I can't tell if this one didn't get submitted properly in my previous review or if you decided not to use it. If the latter, ignore this one too. :)